src/templates/chapter-new.js

```diff
@@ -0,0 +1,500 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
Where is the chapter-new template being used?

Reply:
Not being used right now. Was trying it out for experimentation earlier.